src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -116,6 +122,14 @@ public MainWindow(Stage primaryStage, Logic logic) {
                 setDarkTheme(pref);
             }
         });
+        btnHide.setOnMouseClicked(event -> {
+            int hidden = pref.getInt("hidden", 0);
+            if (hidden == 0) { //dark change to light
```

Comment:
I think your comment for this part should be //hide sensitive data 
```suggestion
            if (hidden == 0) { //hide sensitive data
```
Similarly for the else part as well